chromaui/archive-storybook
# Switch archive-storybook to depend on storybook instead of @storybook/cli

Fixes AP-3573

## What Changed

We adjusted archive-storybook to depend on storybook instead of `@storybook/cli`.



## How to test


Run the the archive-stroybook command with this package version and confirm the SB loads.

## Change Type





<details>
  <summary>📦 Published PR as canary version: <code>0.0.19--canary.15.989f47c.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromaui/archive-storybook@0.0.19--canary.15.989f47c.0
  # or 
  yarn add @chromaui/archive-storybook@0.0.19--canary.15.989f47c.0
  ```
</details>
